crowdgezwitscher/events/models.py

```diff
@@ -36,3 +36,21 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse('events:detail', kwargs={'pk': self.pk})
+
+    def _is_ready_for_twitter(self):
+        """Checks that all fields required for getting tweets have some value."""
+        return all([self.twitter_account_names, self.twitter_hashtags, self.coverage_start, self.coverage_end])
```

Comment:
Aus dem Code in `build_twitter_search_query` würde ich schließen, dass das eher `return all([self.coverage_start, self.coverage_end]) and any([self.twitter_account_names, self.twitter_hashtags])` sein, oder?
Oder zählt das als Konfiguration? Ich hatte versucht nur die Tweets von einem Account zu holen, ohne nach Hashtags zu filtern.